app/helpers/alchemy/admin/elements_helper.rb

```diff
@@ -105,8 +105,8 @@ def element_array_for_options(e, object_method, cell = nil)
       #
       def update_essence_select_elements(page, element)
         elements = page.elements.not_trashed.joins(:contents)
-          .where("alchemy_contents.element_id != #{element.id}")
-          .where("alchemy_contents.essence_type" => "Alchemy::EssenceSelect")
+          .where("#{Content.table_name}.element_id != #{element.id}")
```

Comment:
We shouldn't use dynamic parts in where clauses without escaping them. Rails automatically escapes strings, if we use the Hash or Array syntax:

```
where(["#{Content.table_name}.element_id != ?", element.id])
```